docs/quickstarts/aws.md

```diff
@@ -121,6 +121,13 @@ guide](../configuration-reference/platforms/aws.md).
 
 ### Step 5: Create Lokomotive cluster
 
+Lokoctl uses Terraform behind the scene and it needs SSH access to controller node. For SSH to work add SSH private key corresponding to the public keys specified earlier. Add your SSH private key to `ssh-agent`.
```

Comment:
```suggestion
Lokoctl uses Terraform behind the scene and it needs SSH access to controller nodes. For SSH to work add at least one SSH private key corresponding to the public keys specified earlier to your `ssh-agent`.
```

Reply:
Spelling it controller node is intentional. Since it needs access to just one node, the zeroth one. After that bootkube does it's magic, isn't it how it works?

Reply:
We need SSH access to all controller nodes, to copy secrets: https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/ssh.tf#L3